throttle.js

```diff
@@ -46,9 +46,9 @@ export default function (delay, callback, options) {
 	}
 
 	// Function to cancel next exec
-	function cancel() {
+	function cancel(options) {
```

Comment:
@stamoern could you please implement same check as in https://github.com/niksy/throttle-debounce/blob/203deec24d78abf50751a48c10d0ca7ab1896806/throttle.js#L25-L29

Reply:
Done